src/components/AlertRuleForm.tsx

```diff
@@ -212,49 +217,7 @@ export const AlertRuleForm = ({ rule, onSubmit }: Props) => {
             <div className={styles.expressionContainer}>
               <Label>Expression</Label>
               <HorizontalGroup align="center" wrap marginHeight={0}>
-                <span className={styles.inlineText}>Checks with a sensitivity level of</span>
-                <div className={styles.selectInput}>
-                  <Controller
-                    render={({ field }) => <Select {...field} options={ALERT_SENSITIVITY_OPTIONS} />}
-                    control={control}
-                    name="sensitivity"
-                  />
-                </div>
-                <span className={styles.inlineText}>will fire an alert if less than </span>
-                <Field
-                  invalid={Boolean(errors?.probePercentage)}
-                  error={errors?.probePercentage?.message?.toString()}
-                  className={styles.noMargin}
-                >
-                  <Input
-                    className={styles.numberInput}
-                    {...register('probePercentage', { required: true, max: 100, min: 1 })}
-                    type="number"
-                    data-testid="probePercentage"
-                    id="alertProbePercentage"
-                  />
-                </Field>
-                <span className={styles.inlineText}>% of probes report connection success for</span>
-                <Field
-                  invalid={Boolean(errors?.timeCount)}
-                  error={errors?.timeCount?.message?.toString()}
-                  className={styles.noMargin}
-                >
-                  <Input
-                    {...register('timeCount', { required: true, min: 1, max: 999 })}
-                    data-testid="timeCount"
-                    type="number"
-                    className={styles.numberInput}
-                    id="alertTimeCount"
-                  />
-                </Field>
-                <div className={styles.selectInput}>
-                  <Controller
-                    render={({ field }) => <Select {...field} options={TIME_UNIT_OPTIONS} />}
-                    control={control}
-                    name="timeUnit"
-                  />
-                </div>
+                {expressionContent(styles, control, errors, register)}
```

Comment:
If you end up switching to one of the other approaches I mentioned above, you could change this to be just 
```
{children}
```
or 
```
<SubForm ... />
```

Reply:
Fixed